src/bin/bat/clap_app.rs

```diff
@@ -509,6 +509,17 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                 .hidden_short_help(true)
                 .help("Show diagnostic information for bug reports.")
         )
+        .arg(
+            Arg::with_name("ignored-suffix")
+                .number_of_values(1)
+                .multiple(true)
+                .takes_value(true)
+                .long("ignored-suffix")
```

Comment:
I think we should consider making this `.hidden_short_help(true)`, such that we only show it on `bat --help`, but not for `bat -h` (which should only show the most commonly used options).

Reply:
Thanks @sharkdp for explanation.
It's updated.